src/cmd/ship.go

```diff
@@ -16,10 +16,11 @@ import (
 )
 
 type shipConfig struct {
-	BranchToShip  string
-	InitialBranch string
+	BranchToShip  string // the name of the branch to ship
+	InitialBranch string // the name of the branch that was checked out when running this command
```

Comment:
Again, this doesn't add much to me and I would rather we update the variable name if it will make this clear enough to not need a comment